### PR TITLE
Added POST support for all endpoints

### PR DESCRIPTION
**Pull request by kbagchiGWC**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,6 +1,5 @@
 export ACCOUNT_SID=
 export API_KEY=
 export API_SECRET=
-export OUTGOING_APP_SID=
```
Comment (idelgado) [inline .env.example]: The Java source still  references `OUTGOING_APP_SID`

File: src/main/java/com/twilio/Webapp.java [hunk 2]
```diff
@@ -44,22 +47,24 @@ public static void main(String[] args) throws Exception {
...
-
-            return token.toJwt();
+        /**
+         * Creates an access token with VoiceGrant using your Twilio credentials.
+         *
+         * @returns The Access Token string
+         */
+        post("/accessToken", (request, response) -> {
+            // Read the identity param provided
+            String id = null;
```
Comment (idelgado) [inline src/main/java/com/twilio/Webapp.java]: Is this being sent as `application/json` content-type and is that the content-type we are explicitly supporting?

Comment (idelgado) [inline src/main/java/com/twilio/Webapp.java]: `String identity`

File: src/main/java/com/twilio/Webapp.java [hunk 3]
```diff
@@ -113,50 +121,119 @@ public static void main(String[] args) throws Exception {
...
 
-            VoiceResponse voiceResponse;
-            Say say = new Say.Builder("Congratulations! You have received your first inbound call! Good bye.").build();
+    private static String call(final String to) {
+        VoiceResponse voiceResponse;
+        String toXml = null;
+        if (to == null || to.isEmpty()) {
+            Say say = new Say.Builder("Congratulations! You have made your first call! Good bye.").build();
             voiceResponse = new VoiceResponse.Builder().say(say).build();
+        } else if (Character.isDigit(to.charAt(0)) || to.charAt(0) == '+') {
```
Comment (idelgado) [inline src/main/java/com/twilio/Webapp.java]: Check the whole string: https://github.com/twilio/voice-quickstart-server-python/pull/29/commits/bbf152aab54e9c9c63754af098d6bca95471a182

Added a test for this.

File: src/main/java/com/twilio/Webapp.java [hunk 4]
```diff
@@ -113,50 +121,119 @@ public static void main(String[] args) throws Exception {
...
+        });
 
-            // Print the call SID (a 32 digit hex like CA123..)
-            System.out.println(call.getSid());
+        /**
+         * Makes a call to the specified client using the Twilio REST API.
+         *
+         * @returns The CallSid
+         */
+        post("/placeCall", (request, response) -> {
```
Comment (idelgado) [inline src/main/java/com/twilio/Webapp.java]: Your `/placeCall` has more functionality than the Python one. Should it? If so we should add tests that validate this

File: src/main/java/com/twilio/Webapp.java [hunk 5]
```diff
@@ -170,4 +272,32 @@ private static void dotenv() throws Exception {
...
+        Map<String, String> map = new HashMap<>();
+        for (int i = 0; i < pairs.size(); i++) {
+            NameValuePair pair = pairs.get(i);
+            System.out.println("NameValuePair - name=" + pair.getName() + " value=" + pair.getValue());
+            map.put(pair.getName(), pair.getValue());
+        }
+        return map;
+    }
+
+    private static boolean isPhoneNumber(String s) {
```
Comment (idelgado) [inline src/main/java/com/twilio/Webapp.java]: One minor issue in both Java and Node. If the string is only 1 character this may result in an exception since you are assuming there will be a substring starting at index 1

This was the reason for adding the length check: https://github.com/twilio/voice-quickstart-server-python/blob/master/server.py#L95

It's kind of small corner case but possible. Maybe we should add a test that validates this. You can use identity 'a'.
